Formula/testkube.rb

```diff
@@ -26,6 +26,9 @@ def install
 
     system "go", "build", *std_go_args(output: bin/"kubectl-testkube", ldflags: ldflags),
       "cmd/kubectl-testkube/main.go"
+
+    bin.install_symlink "kubectl-testkube" => "tk"
+    bin.install_symlink "kubectl-testkube" => "testkube"
```

Comment:
If we wanted to call the binary `testkube`, why did we override `output` in `std_go_args` above?

Reply:
HI @carlocab we want to call kubectl-testkube too as it's usually kubectl plugin